analyses/gene-set-enrichment-analysis/run-gsea.sh

```diff
@@ -41,8 +41,27 @@ INPUT_FILE="${DATA_DIR}/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds"
 OUTPUT_FILE="${RESULTS_DIR}/gsva_scores_stranded.tsv"
 Rscript --vanilla 01-conduct-gsea-analysis.R --input ${INPUT_FILE} --output ${OUTPUT_FILE}
 
+
 if [[ "$RUN_FOR_SUBTYPING" -lt "1" ]]; then
   ######## Model GSVA scores ############
   # Only run when pbta-histologies.tsv is generated which has harmonized_diagnosis
   Rscript -e "rmarkdown::render('02-model-gsea.Rmd', clean = TRUE, params=list(is_ci = ${IS_CI}))"
 fi
+
+
+######## Calculate scores from stranded expression data for threshold-passing tumors only #######
+INPUT_FILE="${DATA_DIR}/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds"
+OUTPUT_FILE="${RESULTS_DIR}/gsva_scores_stranded_thresholded.tsv"
+Rscript --vanilla 01-conduct-gsea-analysis.R --input ${INPUT_FILE} --output ${OUTPUT_FILE} --apply_tumor_purity_threshold
+
+
+# Assess results generated for threshold-passing tumors
+Rscript -e "rmarkdown::render('03-assess-gsea-at-threshold.Rmd', clean = TRUE)"
```

Comment:
This requires modeling that relies on having cancer group labels in place, correct? Wouldn't we want this to be within the same logic as `02-model-gsea.Rmd` above in that case?

Reply:
I'm not sure I follow what you mean by "cancer group labels in place"? The `03-assess-gsea-at-threshold.Rmd` notebook actually performs its relevant modeling directly rather than reading in anything that would have been produced from `02-model-gsea.Rmd`. The main reason I made that choice is because we want this analysis to focus on the grouping `cancer_group_display`, not `cancer_group`, and I didn't want to add all kinds of wrangling to change that out in the `02-model-gsea.Rmd` notebook.

Reply:
Right, but how an individual sample gets grouped into `cancer_group_display` is based on it's `cancer_group` label. A `cancer_group` label is subject to change during the subtyping process. So if you have an analysis that's in any way reliant on the `cancer_group` label (which this is), you'd only want that part to run when you are not subtyping.

Reply:
In https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1659/commits/7994f234ad427fb63e70b7feb143f551ea56ea07 I updated the logic in `run-gsea.sh` accordingly to now only run `cancer_group`-consuming items if we are NOT subtyping.

In https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1659/commits/7994f234ad427fb63e70b7feb143f551ea56ea07 and https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1659/commits/b909aaea43badf14f19bc95d527965bce19de8aa, I cleaned up some related comments in that `run-gsea.sh` script as well.